### ISSUE TITLE
Adding an option to disable the auto-install of updates when no users are logged in. 

### ISSUE THREAD

**Issue by cesmith17** [Macjutsu/super#234]
Hello! 

I've noticed during a couple of situations like the deployment of SUPER via Jamf where there aren't users signed in, that SUPER will auto-install updates which has caused some of our users to be confused since they were expecting the defer window to pop after coming back to the computer and signing in to see the prompt.

Would it be possible to add this option into SUPER, where we can disable the auto-install when the "loginwindow" user is detected or disable updates when the loginwindow is present 

Thanks!

**Comment by Macjutsu** [comment]
Good idea... may have to wait a bit on this one though as there a few major issues I'm working on for the initial release of v5.0.0.

**Comment by Macjutsu** [comment]
Sorry for not closing this earlier... this feature was added to `super`. Please try the latest build here https://github.com/Macjutsu/super/releases/tag/v5.1.0-rc5
